runtime/centrifuge/src/migrations.rs

```diff
@@ -54,13 +51,16 @@ pub type UpgradeCentrifuge1025 = (
 		LocalCurrencyIdUsdc,
 	>,
 	// Register new canonical USDC on Celo
-	runtime_common::migrations::update_celo_usdcs::Migration<super::Runtime>,
-	// Init local representation for all assets
+	runtime_common::migrations::update_celo_usdcs::AddNewCeloUsdc<super::Runtime>,
+	// Update custom metadata by initiating local representation for all assets
 	runtime_common::migrations::local_currency::translate_metadata::Migration<
 		super::Runtime,
 		UsdcVariants,
 		LocalAssetIdUsdc,
 	>,
+	// Change name and symbol of Celo Wormhole USDC
+	// NOTE: Needs to happen after metadata translation because expects new CustomMetadata
+	runtime_common::migrations::update_celo_usdcs::UpdateWormholeUsdc<super::Runtime>,
```

Comment:
I had to split both migrations because we need the new non-Wormhole Celo USDC variant for translating the asset metadata to the new custom metadata which includes the `local_representation` field. The name/symbol update migration for the existing Wormhole Celo USDC variant must happen after translating our metadata, else the storage retrieval fails without defining a `storage_alias` with the old custom metadata.